sdk/communication/azure-communication-identity/src/test/java/com/azure/communication/identity/CommunicationLoggerPolicy.java

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.communication.identity;
+
+import com.azure.core.http.HttpPipelineCallContext;
+import com.azure.core.http.HttpPipelineNextPolicy;
+import com.azure.core.http.HttpResponse;
+import com.azure.core.http.policy.HttpPipelinePolicy;
+import com.azure.core.util.logging.ClientLogger;
+
+import reactor.core.publisher.Mono;
+
+/**
+ * A policy that logs the Http Response
+ */
+public class CommunicationLoggerPolicy implements HttpPipelinePolicy {
```

Comment:
what do we use this for?

Reply:
This is something I added to log the ms-cv headers for our live tests. We have plans of removing this in the future and replacing it with a lambda (https://github.com/Azure/azure-sdk-for-java/pull/18786/files) However, this will cause conflicts with our current work and I so was planning to do it after the API changes.

Reply:
This is for printing out MS-CV header

Reply:
I am curious if this LoggerPolicy will be only used in our test code only?